src/plugins/Log.ts

```diff
@@ -90,14 +95,34 @@ class Log {
         }
     }
 
-    reportLog = (logInfo: any) => {
+    reportLog(logInfo: any): void {
         const logData = new LogData();
         const activeSpan = this.tracer ? this.tracer.getActiveSpan() : undefined;
         const spanId = activeSpan ? activeSpan.spanContext.spanId : '';
         logData.initWithLogDataValues(this.logData, spanId, logInfo);
 
         this.logs.push(logData);
     }
+
+    shimConsole(): void {
+        ConsoleShimmedMethods.forEach((method) => {
+            const consoleReference: any = console;
+            const originalConsoleMethod = consoleReference[method].bind(console);
+
+            consoleReference[method] = (...args: any[]) => {
+                const logInfo = {
+                    logMessage:  util.format.apply(util, args),
+                    logLevel: method ? method.toUpperCase() : 'INFO',
+                    logLevelCode: logLevels[method],
+                    logContextName: ConsoleLogContext,
```

Comment:
I think, `ConsoleLogContext` can be `STDOUT` for non-error logs, and `STDERR` for error logs.

Reply:
Done